integration-tests/smoke/auto_ocr_test.go

```diff
@@ -173,6 +227,471 @@ var _ = Describe("Automation OCR Suite @auto-ocr", func() {
+						"than initial counter which was %s, but got %s", initialCounters[i], counter)
+				}
+			}, "5m", "1s").Should(Succeed())
+		}
+
+		if testToRun == PauseRegistryTest {
+			By("pauses the registry and makes sure that the upkeeps are no longer performed")
+			// Observe that the upkeeps which are initially registered are performing
+			Eventually(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					counter, err := consumers[i].Counter(context.Background())
+					g.Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(counter.Int64()).Should(BeNumerically(">", int64(0)),
+						"Expected consumer counter to be greater than 0, but got %d")
+				}
+			}, "5m", "1s").Should(Succeed())
+
+			// Pause the registry
+			err := registry.Pause()
+			Expect(err).ShouldNot(HaveOccurred(), "Could not pause the registry")
+			err = chainClient.WaitForEvents()
+			Expect(err).ShouldNot(HaveOccurred(), "Failed to wait for events")
+
+			// Store how many times each upkeep performed once the registry was successfully paused
+			var countersAfterPause = make([]*big.Int, len(upkeepIDs))
+			for i := 0; i < len(upkeepIDs); i++ {
+				countersAfterPause[i], err = consumers[i].Counter(context.Background())
+				Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+					" for upkeep at index "+strconv.Itoa(i))
+			}
+
+			// After we paused the registry, the counters of all the upkeeps should stay constant
+			// because they are no longer getting serviced
+			Consistently(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					latestCounter, err := consumers[i].Counter(context.Background())
+					Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(latestCounter.Int64()).Should(Equal(countersAfterPause[i].Int64()),
+						"Expected consumer counter to remain constant at %d, but got %d",
+						countersAfterPause[i].Int64(), latestCounter.Int64())
+				}
+			}, "1m", "1s").Should(Succeed())
+		}
+
+		if testToRun == HandleKeeperNodesGoingDown {
+			By("upkeeps are still performed if some keeper nodes go down")
+			var initialCounters = make([]*big.Int, len(upkeepIDs))
+
+			// Watch upkeeps being performed and store their counters in order to compare them later in the test
+			Eventually(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					counter, err := consumers[i].Counter(context.Background())
+					initialCounters[i] = counter
+					g.Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(counter.Int64()).Should(BeNumerically(">", int64(0)),
+						"Expected consumer counter to be greater than 0, but got %d", counter.Int64())
+				}
+			}, "5m", "1s").Should(Succeed())
```

Comment:
Did you dig into why we need 5 minutes? I see we only require it to be performed once